services/policies/pkg/config/config.go

```diff
@@ -0,0 +1,68 @@
+package config
+
+import (
+	"context"
+	"github.com/owncloud/ocis/v2/ocis-pkg/shared"
+	"time"
+)
+
+// Config combines all available configuration parts.
+type Config struct {
+	Commons           *shared.Commons       `yaml:"-"` // don't use this directly as configuration for a service
+	GRPC              GRPC                  `yaml:"grpc"`
+	Service           Service               `yaml:"-"`
+	TokenManager      *TokenManager         `yaml:"token_manager"`
+	Events            Events                `yaml:"events"`
+	Reva              *shared.Reva          `yaml:"reva"`
+	GRPCClientTLS     *shared.GRPCClientTLS `yaml:"grpc_client_tls"`
+	MachineAuthAPIKey string                `yaml:"machine_auth_api_key" env:"OCIS_MACHINE_AUTH_API_KEY;POLICIES_MACHINE_AUTH_API_KEY" desc:"Machine auth API key used to validate internal requests necessary for the access to resources from other services."`
+	Context           context.Context       `yaml:"-"`
+	Log               *Log                  `yaml:"log"`
+	Engine            Engine                `yaml:"engines"`
+	Postprocessing    Postprocessing        `yaml:"postprocessing"`
+}
+
+// Service defines the available service configuration.
+type Service struct {
+	Name string `yaml:"-"`
+}
+
+// GRPC defines the available grpc configuration.
+type GRPC struct {
+	Addr      string                 `ocisConfig:"addr" env:"POLICIES_GRPC_ADDR" desc:"The bind address of the GRPC service."`
+	Namespace string                 `ocisConfig:"-" yaml:"-"`
+	TLS       *shared.GRPCServiceTLS `yaml:"tls"`
+}
+
+// TokenManager is the config for using the reva token manager
+type TokenManager struct {
+	JWTSecret string `yaml:"jwt_secret" env:"OCIS_JWT_SECRET;POLICIES_JWT_SECRET" desc:"The secret to mint and validate jwt tokens."`
+}
+
+// Engine configures the policy engine.
+type Engine struct {
+	Timeout  time.Duration `yaml:"timeout" env:"POLICIES_ENGINE_TIMEOUT" desc:"Sets the timeout."`
+	Policies []string      `yaml:"policies"`
+}
+
+// Postprocessing defines the config options for the postprocessing policy handling.
+type Postprocessing struct {
+	Query string `yaml:"query" env:"POLICIES_POSTPROCESSING_QUERY" desc:"Sets the postprocessing query."`
```

Comment:
```suggestion
	Query string `yaml:"query" env:"POLICIES_POSTPROCESSING_QUERY" desc:"Defines the 'Complete Rules' variable defined in the rego rule set this step uses for its evaluation. Defaults to deny if the variable was not found."`
```
The naming is derived from: https://www.openpolicyagent.org/docs/latest/#complete-rules
`Pls check that if there is no match, the evaluation defaults to deny!`
Possibly a log entry if no match was found so that admins can check if they mistyped the name.